app/assets/stylesheets/content/_buttons.sass

```diff
@@ -63,6 +63,9 @@ a.button
   &:hover
     text-decoration: none
 
+  &.-with-gap
+    margin-top: 1rem
```

Comment:
What makes me a little nervous is that this rule is more specific than it seems.

Gap? Of course on top... Gaps to the left would not make sense, would they?

But then I am irritated about buttons anyway. Apparently it makes sense that they only have margins to the right and at the bottom.